### DISCUSSION TITLE
4D023002-8F85-44E8-96D2-812D97057DE2

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#159] [General]
# Star Barycenter

An addon that has a barycenter with every spectral type of brown dwarf and main sequence star ranging from Y9 to L0 for the brown dwarves and M9V to O0V.

https://celestia.mobi/addon?item=4D023002-8F85-44E8-96D2-812D97057DE2

**Comment by exocritic5457** [comment]
Interesting how if you speed it up enough the stars start forming straight lines

**Comment by AU-Microscoppi-b** [comment]
What is the name of the system (tryna find it)

**Comment by zjx29rjhdh-pixel** [comment]
Very beautiful… Not WHAT I EXPECTED

**Comment by LangLangFCS** [comment]
Wow It's amazing!
